daily-check.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 日常巡检
+summary: 介绍 TiDB 集群需要常关注的性能指标。
+category: reference
```

Comment:
Please add alias. @toutdesuite

Reply:
updated in https://github.com/pingcap/docs-cn/pull/3613